tests/test_hooks/test_cmd.py

```diff
@@ -147,14 +147,21 @@ def test_hook_writes_to_stderr(stack, capfd):
 def test_default_shell_is_sh(stack, capfd):
     Cmd("echo $0", stack).run()
     cap = capfd.readouterr()
-    assert cap.out.strip() == "/bin/sh"
+    assert cap.out.strip().split("/")[-2:] == ["bin", "sh"]
```

Comment:
Would it be better here to instead use a mock so that we can control the response?